example_codes/agent_vs_keyboard_example.py

```diff
@@ -1,3 +1,11 @@
+#
+# Controlls: MOVE: W A S D 
+#            HIT: right arrow
+#            BLOCK: left arrow
+#            KICK LOW: up arrow
```

Comment:
Up and Down keys are possibly mixed up? See lines 47 and 48

Reply:
They were, thanks for mentioning.